code/modules/mob/dead/observer/orbit.dm

```diff
@@ -94,9 +94,7 @@
 
 					// Not-very-datumized antags follow
 					// Associative list of antag name => whether this mind is this antag
-					var/other_antags = list(
-						"Changeling" = (mind.changeling != null)
-					)
+					var/list/other_antags = list()
```

Comment:
🎉 
but also, might aswell remove this list now.

Reply:
This is still used by other non datum antags below, like cult, blob, etc.